docs/formats/zson.md

```diff
@@ -424,7 +424,8 @@ value in a record, as an element in an array, etc).
 #### 3.3.1 String Escape Rules
 
 Double-quoted `string` syntax is the same as that of JSON as described
-[RFC 8259](https://tools.ietf.org/html/rfc8259#section-7), specifically:
+[RFC 8259](https://tools.ietf.org/html/rfc8259#section-7), except for
+the special case `\/`, specifically:
```

Comment:
https://tools.ietf.org/html/rfc8259#section-7 specifically says `\/` is a recognized two-character sequence escape representation for `/`, so this isn't an exception.

Reply:
Thanks for checking.  Mystery solved!